### ISSUE TITLE
Window not rendered without resize (on macOS)

### ISSUE THREAD

**Issue by eruvanos** [pvcraven/arcade#299]
## Bug Report
Starting the given example opens a window, which is not getting rendered.
Can be fixed by manual resizing or using the commented codeline

### Actual behavior:
Window without text.

### Expected behavior:
Window shows `Hello`

### Steps to reproduce/example code:
Run example:
```
import arcade

class Demo(arcade.Window):
    def __init__(self):
        super().__init__(resizable=True)
        #  self.width = self.width - 1 # triggers resize by code, fix macOS black screen

    def on_draw(self):
        arcade.start_render()
        arcade.draw_text('hello', 50, 50, arcade.color.WHITE)

d = Demo()
arcade.run()
```

### Infos

OS: macOS Mojave

```
22:14 $ python --version
Python 3.7.2
```

```
Using pip3...
Package    Version
---------- -------
arcade     1.3.7  
future     0.17.1 
Pillow     5.4.1  
pip        19.0.1 
pyglet     1.3.2  
setuptools 40.7.2 
wheel      0.32.3 
```

**Comment by eruvanos** [comment]
Still not working, furthermore after resizing the rendered window shows up much smaller.

**Comment by pvcraven** [comment]
It renders for me, although on window resize it moves the text. Which is a problem.

**Comment by pvcraven** [comment]
Closed with 2.0 release. Reopen if still an issue please.
